youtube_dl/extractor/youtube.py

```diff
@@ -353,7 +353,7 @@ class YoutubeIE(YoutubeBaseInfoExtractor):
         r'(?:www\.)?invidious\.13ad\.de',
         r'(?:www\.)?invidious\.mastodon\.host',
         r'(?:www\.)?invidious\.zapashcanon\.fr',
-        r'(?:www\.)?invidious\.kavin\.rocks',
+        r'(?:www\.)?(invidious(-us)?|piped)\.kavin\.rocks',
```

Comment:
Do not capture groups you don't use.

Reply:
?

Reply:
He means you are supposed to make it a non-capturing group. Like this:
`(?:invidious(?:-us)?|piped)`